pkg/hooks/audit_logger_sqlite.go

```diff
@@ -0,0 +1,137 @@
+package hooks
+
+import (
+	"database/sql"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/util"
+)
+
+type SQLiteAuditLogger struct {
+	sqlite *sql.DB
```

Comment:
it probably doesn't matter that much but there should probably be a `Close()` function to call `sqlite.Close()`